code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -0,0 +1,39 @@
+#define ASH_WALKER_SPAWN_THRESHOLD 2
```

Comment:
I'm not sure why this is in its own file either come to think of it, why scatter ash walker stuff across the codebase?